server/entity/src/lib.rs

```diff
@@ -1,4 +1,10 @@
 pub mod account_activation;
+pub mod currency_type;
+pub mod expense;
+pub mod predefined_expense;
+pub mod recurrence_type;
+pub mod transaction;
 pub mod user;
 
 pub type Id = i64;
+pub type Value = i64;
```

Comment:
Is this a rust convention? _Value_ is too abstract for a number type

Reply:
Type aliases are always encouraged to make the code more readable by making connection between the types of things. Two fields in different structs that are `i64` typed don't say much, but if they both have the same type alias then it is sure that there is a connection between them. Also it helps if you need to change a type, you only have to change it in one place.

These were my reasonings, Value is the type for the Value field in our structs which is the moni type. I don't know what type it really should be because in some jsons it was a floating point value, and in others it was an integer, and somewhere I've seen string but that might have just been a hallucination :smile: . So you could help with that here or in you own PR and when it is clear, we will update this type alias, and the columns migrations easily.

But yeah, value is a bit too abstract, maybe `Money`? or any other ideas?

Reply:
I kind of guessed the first half. 

Yep, we have different types for "values" throughout our data model.

Maybe `Amount`, but it looks like `Money` is a good as well?

Reply:
So it remains and integer for now and let's settle on MoneyAmount 😄